hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -0,0 +1,1332 @@
+import com.hedera.hapi.node.transaction.Query;
+import com.hedera.hapi.node.transaction.TransactionBody;
+import com.hedera.node.app.service.mono.legacy.core.jproto.JKey;
+import com.hedera.node.app.spi.key.HederaKey;
+import com.hedera.pbj.runtime.Codec;
+import com.hedera.pbj.runtime.io.buffer.BufferedData;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import com.hedera.pbj.runtime.io.stream.ReadableStreamingData;
+import com.hedera.pbj.runtime.io.stream.WritableStreamingData;
+import com.hederahashgraph.api.proto.java.AccountID.AccountCase;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Objects;
+import java.util.Optional;
+
+public final class PbjConverter {
+    public static @NonNull AccountID toPbj(@NonNull com.hederahashgraph.api.proto.java.AccountID accountID) {
+        requireNonNull(accountID);
+        final var builder =
+                AccountID.newBuilder().shardNum(accountID.getShardNum()).realmNum(accountID.getRealmNum());
+        if (accountID.getAccountCase() == AccountCase.ALIAS) {
+            builder.alias(Bytes.wrap(accountID.getAlias().toByteArray()));
+        } else {
+            builder.accountNum(accountID.getAccountNum());
+        }
+        return builder.build();
+    }
+
+    public static @NonNull TokenID toPbj(@NonNull com.hederahashgraph.api.proto.java.TokenID tokenID) {
+        requireNonNull(tokenID);
+        return TokenID.newBuilder()
+                .shardNum(tokenID.getShardNum())
+                .realmNum(tokenID.getRealmNum())
+                .tokenNum(tokenID.getTokenNum())
+                .build();
+    }
+
+    public static @NonNull TransactionBody toPbj(@NonNull com.hederahashgraph.api.proto.java.TransactionBody txBody) {
+        requireNonNull(txBody);
+        try {
+            final var bytes = txBody.toByteArray();
+            final var ret = TransactionBody.PROTOBUF.parse(BufferedData.wrap(bytes));
+            return ret;
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static @NonNull Key toPbj(@NonNull com.hederahashgraph.api.proto.java.Key keyValue) {
+        requireNonNull(keyValue);
+        try {
+            final var bytes = keyValue.toByteArray();
+            return Key.PROTOBUF.parse(BufferedData.wrap(bytes));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static @NonNull com.hederahashgraph.api.proto.java.Key fromPbj(@NonNull Key keyValue) {
+        requireNonNull(keyValue);
+        try {
+            final var bytes = asBytes(Key.PROTOBUF, keyValue);
+            return com.hederahashgraph.api.proto.java.Key.parseFrom(bytes);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
message is missing + IllegalArgementException might make more sense for all the converter methods